src/Controller/HoldRequestController.php

```diff
@@ -45,34 +46,49 @@ class HoldRequestController extends ServiceController
      *     @SWG\Response(
      *         response="404",
      *         description="Not found",
-     *         @SWG\Schema(ref="#/definitions/ErrorResponse")
+     *         @SWG\Schema(ref="#/definitions/HoldRequestErrorResponse")
      *     ),
      *     @SWG\Response(
      *         response="500",
      *         description="Generic server error",
-     *         @SWG\Schema(ref="#/definitions/ErrorResponse")
+     *         @SWG\Schema(ref="#/definitions/HoldRequestErrorResponse")
      *     ),
      *     security={
      *         {
      *             "api_auth": {"openid offline_access api"}
```

Comment:
You might want to update the documentation to include the actual scopes that are required. This is finally becoming meaningful! 😄